crates/conjure_core/src/solvers.rs

```diff
@@ -2,32 +2,32 @@ use strum_macros::Display;
 use strum_macros::{EnumIter, EnumString};
 
 /// All supported solvers.
-#[derive(Debug, EnumString, EnumIter, Display)]
-pub enum Solver {
+#[derive(Debug, EnumString, EnumIter, Display, PartialEq, Eq, Hash, Clone, Copy)]
+pub enum SolverName {
     Minion,
     KissSAT,
 }
```

Comment:
I don't think this solver enum is needed or desirable anymore - its just leftover stuff from the older solver interface.

To reduce boilerplate, maybe SolverAdaptor could have a method called get_solver_family()?

Reply:
That way everything about a given solver is defined inside the SolverAdaptor itself: with the current approach, implementers will have to edit this file as well as their adaptor and they may forget, etc.

Reply:
Maybe I am misunderstanding this - is there a purpose of SolverName seperate from both family and adaptor?

Reply:
> Maybe I am misunderstanding this - is there a purpose of SolverName separate from both family and adaptor?

It represents a specific solver that we can target. So, not sure - does SolverAdaptor do that too?

I could try to do that but:

- Solver adaptors are part of conjure_oxide so I might not be able to use them in my context definition
- I still want the family to be able to give a list of its member solvers - this is used for resolving rule sets